index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Aamir Asaram</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <img src="assets/images/menu.png" class="menu-btn"">
+    </nav>
+  </header>
+  <div class="bg">
+    <div class="intro">
+      <div>
+        <h2 class="blue heading">Hey there. I'm Aamir.</h2>
+        <h1 class="green heading">I'm a software developer</h1>
+      </div>
+      <article>
+        <p class="blue">I can help you build a product, feature or website.
+        Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don't hesitate
+        to contact me.</p>
+      </article>
+
+      <div class="social">
+        <ul>
+          <!-- </a> -->
+          <a href="https://twitter.com/aaaaamir_"><li><button class="social-btn"><i class="fa fa-twitter"></i></button></li></a>
+          <a href="https://www.linkedin.com/in/aamir-asaram-089802213/"><li><button class="social-btn"><i class="fa fa-linkedin"></i></button></li></a>
+          <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-medium"></i></button></li></a>
+          <a href="https://github.com/aamir-asaram"><li><button class="social-btn"><i class="fa fa-github"></i></button></li></a>
+          <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-angellist"></i></button></li></a>
+        </ul>
```

Comment:
- [ ] To match the original design from Figma, I think it would be better of you improve your current design **of your social media icons** by using the correct `size`, `height`, `width`, `spacing`, between,... If you take a look on your design and the original one you will see a big difference to improve

| Figma Deisgn | Your Design |
| -------------- | ----------------- |
| ![Screenshot from 2023-04-27 08-07-32](https://user-images.githubusercontent.com/83042742/234774136-82c69d7c-8bee-48cd-a86c-518670a15d0f.png) | ![Screenshot from 2023-04-27 08-05-49](https://user-images.githubusercontent.com/83042742/234773771-95816b4a-6215-4842-a279-cea0a9f56821.png) |